packages/components/notification/README.md

```diff
@@ -0,0 +1,4 @@
+# @contentful/f36-notification
```

Comment:
Do we need to add this in every v4 components?

Reply:
Yes, because all the components are published as independent packages, so it needs the README on all of them to have this disclaimer

Reply:
I think we need to update the plop template

My branch only has one README with all the documentation, but I guess every package needs a `README.md` and `Component.mdx`, right?

Reply:
Yeah, I think that it got updated to have the disclaimer, but not to have both files